### PR TITLE
Add variant for Elecrow CrowPanel 7.0"

### PR DESCRIPTION
**Pull request by lboue**
### Checklist
1. [x] Please provide specific title of the PR describing the change, including the component name (*eg. „Update of Documentation link on Readme.md“*)
2. [x] Please provide related links (*eg. Issue which will be closed by this Pull Request*)
3. [x] Please **update relevant Documentation** if applicable
4. [x] Please check [Contributing guide](https://docs.espressif.com/projects/arduino-esp32/en/latest/contributing.html)
5. [x] Please **confirm option to "Allow edits and access to secrets by maintainers"** when opening a Pull Request

*This entire section above can be deleted if all items are checked.*

-----------
## Description of Change
Add variant for Elecrow CrowPanel 7.0"
* feat(esp32): Add variant for Elecrow CrowPanel 7.0"

Variant for https://www.elecrow.com/esp32-display-7-inch-hmi-display-rgb-tft-lcd-touch-screen-support-lvgl.html

## Tests scenarios
I have tested my Pull Request on Arduino-esp32 core 3.0.1 with Elecrow CrowPanel 7.0" with this scenario.

## Related links
Fixes: #9895

### CODE REVIEW

**Comment by lucasssvaz** [comment]
Just as a heads up, you dont need to close and open a new PR to update your files. Just push to the base branch that the pull request refers to.

File: boards.txt [hunk 1]
```diff
@@ -37110,3 +37110,157 @@ walter.menu.EraseFlash.all=Enabled
...
+elecrow_crowpanel_7.menu.FlashMode.dio.build.flash_mode=dio
+elecrow_crowpanel_7.menu.FlashMode.dio.build.boot=dio
+
+elecrow_crowpanel_7.menu.FlashFreq.80=80MHz
+elecrow_crowpanel_7.menu.FlashFreq.80.build.flash_freq=80m
+elecrow_crowpanel_7.menu.FlashFreq.40=40MHz
+elecrow_crowpanel_7.menu.FlashFreq.40.build.flash_freq=40m
+
+elecrow_crowpanel_7.menu.FlashSize.16M=4MB (32Mb)
+elecrow_crowpanel_7.menu.FlashSize.16M.build.flash_size=4MB
```
Comment (P-R-O-C-H-Y) [inline boards.txt:37231]: Please rename the FlashSize option to 4M, or you can delete it completely. As the board is sold with 4MB Flash chip. 
```suggestion
elecrow_crowpanel_7.menu.FlashSize.4M=4MB (32Mb)
elecrow_crowpanel_7.menu.FlashSize.4M.build.flash_size=4MB
```

File: boards.txt [hunk 2]
```diff
@@ -37110,3 +37110,157 @@ walter.menu.EraseFlash.all=Enabled
...
+elecrow_crowpanel_7.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+elecrow_crowpanel_7.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+elecrow_crowpanel_7.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+elecrow_crowpanel_7.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+elecrow_crowpanel_7.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FAT)
+elecrow_crowpanel_7.menu.PartitionScheme.fatflash.build.partitions=ffat
+elecrow_crowpanel_7.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+elecrow_crowpanel_7.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9MB FATFS)
+elecrow_crowpanel_7.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+elecrow_crowpanel_7.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
```
Comment (P-R-O-C-H-Y) [inline boards.txt:37199]: Please remove partitions for 16MB flash.
```suggestion
```

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@lboue Just small nitpicks ptal. Otherwise looking good.

File: boards.txt (hunk 1 above)
Comment (lboue) [reply boards.txt:37231]: @P-R-O-C-H-Y Thanks for the review.

**Comment by lboue** [comment]
> @lboue Just small nitpicks ptal. Otherwise looking good.

It sould be with last commit.

**Pull request merged**